kube-runtime/src/watcher.rs

```diff
@@ -133,7 +133,7 @@ async fn step_trampolined<K: Resource + Clone + DeserializeOwned + Debug + Send
             resource_version,
             mut stream,
         } => match stream.next().await {
-            Some(Ok(WatchEvent::Added(obj))) | Some(Ok(WatchEvent::Modified(obj))) => {
+            Some(Ok(WatchEvent::Added(obj) | WatchEvent::Modified(obj))) => {
```

Comment:
oh, i was wondering where this came from for a while, until i realised it's just the new smarter pattern matching :+1: